src/PKSim.Core/Commands/AddBuildingBlockToProjectCommand.cs

```diff
@@ -23,6 +23,7 @@ protected override void ExecuteWith(IExecutionContext context)
       {
          var project = context.CurrentProject;
          project.AddBuildingBlock(_buildingBlockToAdd);
+         _buildingBlockToAdd.HasChanged = true;
```

Comment:
This is now required since the flag is not set anymore automatically upon loading (problem with multiple conversion message). 
By setting the flag in the AddBuildingBlockToProjectCommand we should be good to go